src/openrct2/ride/Vehicle.cpp

```diff
@@ -9667,20 +9667,9 @@ int32_t Vehicle::UpdateTrackMotion(int32_t* outStation)
         newAcceleration += 511;
     }
     newAcceleration >>= 9;
-    int32_t curAcceleration = newAcceleration;
-    int32_t accelerationDecrease1 = vehicle->velocity;
-    if (accelerationDecrease1 < 0)
-    {
-        accelerationDecrease1 = -accelerationDecrease1;
-        accelerationDecrease1 >>= 12;
-        accelerationDecrease1 = -accelerationDecrease1;
-    }
-    else
-    {
-        accelerationDecrease1 >>= 12;
-    }
 
-    curAcceleration -= accelerationDecrease1;
+    int32_t curAcceleration = newAcceleration;
+    curAcceleration -= vehicle->velocity / 4096;
```

Comment:
is there the same calculation in the mini golf code?

Reply:
Can't say, thats the only place I've found so far that did this.